developer-extension/src/panel/sessionReplayPlayer/startSessionReplayPlayer.ts

```diff
@@ -0,0 +1,167 @@
+import type { BrowserRecord } from '../../../../packages/rum/src/types'
+import { IncrementalSource, RecordType } from '../../../../packages/rum/src/types'
+import { createLogger } from '../../common/logger'
+import { listenSdkMessages } from '../backgroundScriptConnection'
+import type { MessageBridgeUp } from './types'
+import { MessageBridgeDownType } from './types'
+
+const sandboxLogger = createLogger('sandbox')
+
+export type SessionReplayPlayerStatus = 'loading' | 'waiting-for-full-snapshot' | 'ready'
+
+const sandboxParams = new URLSearchParams({
+  staticContext: JSON.stringify({
+    tabId: 'xxx',
+    origin: location.origin,
+    featureFlags: {
+      // Allows to easily inspect the DOM in the sandbox
+      rum_session_replay_iframe_interactive: true,
+
+      // Use the service worker
+      rum_session_replay_service_worker: true,
+      rum_session_replay_service_worker_debug: false,
+
+      rum_session_replay_disregard_origin: true,
+    },
+  }),
+})
+const sandboxOrigin = 'https://session-replay-datadoghq.com'
+const sandboxUrl = `${sandboxOrigin}/0.68.0/index.html?${String(sandboxParams)}`
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should we update this `0.68.0` version from time to time? if yes, we could maybe extract a constant for it

Reply:
Yes but we might have updated the communication API, It might be more work than a search & replace